src/core/Akka/Routing/ConsistentHashRouter.cs

```diff
@@ -292,7 +292,7 @@ public override string ToString()
                 case ActorRefRoutee actorRef:
                     return ToStringWithFullAddress(actorRef.Actor.Path);
                 case ActorSelectionRoutee selection:
-                    return ToStringWithFullAddress(selection.Selection.Anchor.Path) +
+                    return ToStringWithFullAddress(selection.Selection.Anchor.Path).TrimEnd('/') +
```

Comment:
So it was just the trailing slash that created this issue?

Reply:
Yup. It results in double slashes. When everything is remote it works because they all have it. If you have a local routee you start seeing a little weirdness.